docs/src/pages/index.tsx

```diff
@@ -183,33 +259,46 @@ function Features() {
   return (
     <div className="container margin-top--lg margin-bottom--md">
       <Heading as="h1" className="margin-bottom--md text--center text--primary">
-        Features
+        Capabilities
       </Heading>
       <p className={clsx('text--center', styles.sectionSubheading)}>
         Simple, powerful, fast. Pick three.
       </p>
 
       <Feature
-        text="WebGPU APIs"
-        secondaryText="Utilize existing WebGPU-based components or craft your own WGSL shader to achieve any desired effect and directly leverage GPU parallel processing capabilities."
+        text="Batteries included"
+        secondaryText="Process streams, render text, images, GIFs or websites, and combine them into output streams without writing low-level code."
+        image={<GiBattery100 className={styles.featureIcon} style={{ fontSize: 80 }} />}
+      />
+
+      <Feature
+        text="Highly customizable"
+        secondaryText="Create custom effects with WGSL shaders and directly leverage GPU parallel processing capabilities."
+        image={<FaGears className={styles.featureIcon} style={{ fontSize: 80 }} />}
+      />
+
+      <Feature
+        text="WebGPU rendering"
+        secondaryText="Rendering is hardware accelerated using wgpu, a Rust library implementing WebGPU standard. It's fast, modern, and cross-platform."
         image={wgpuLogo}
       />
+
       <Feature
-        text="Real-time processing"
-        secondaryText="Process live video streams in real-time with low latency. Blazingly fast processing makes video conferencing, live-streaming, and broadcasting an everyday bread and butter for Live Compositor."
-        image={<MdLiveTv className={styles.featureIcon} style={{ fontSize: 80 }} />}
-        inverted
+        text="Animated transitions"
+        secondaryText="Dynamically change the composition with customizable animated transitions."
```

Comment:
```suggestion
        secondaryText="Apply scene changes with animated transitions."
```
or maybe `Trigger`, `Run` (not really sure what fits better)
scene<->composition would also be fine, but i think scene is intuitive enough on it's own and this is the term we use in docs